app/actions/stat_created_plan/generate.rb

```diff
@@ -0,0 +1,48 @@
+module Actions
+  module StatCreatedPlan
+    class Generate
+      class << self
+        def full(org)
+          OrgDateRangeable.split_months_from_creation(org) do |start_date, end_date|
+            create_count_for_date(start_date: start_date, end_date: end_date, org: org)  
+          end
+        end
+
+        def last_month(org)
+          months = OrgDateRangeable.split_months_from_creation(org)
+          last = months.last
+          if last.present?
+            create_count_for_date(start_date: last[:start_date], end_date: last[:end_date], org: org)
+          end
+        end
+        
+        def full_all_orgs
+          Org.all.each do |org|
+            full(org)
+          end
+        end
+
+        def last_month_all_orgs
+          Org.all.each do |org|
+            last_month(org)
+          end
+        end
+
+        private
+
+        def count_plans(start_date: , end_date: , org:)
+          users = User.where('users.org_id = ?', org.id)
+          plans = Plan.where('plans.created_at >= ? AND plans.created_at <= ?', start_date, end_date)
+          owner_coowner = Role.where(access: Role.access_values_for(:creator).concat(Role.access_values_for(:administrator)).uniq)
```

Comment:
It’s defined on the development branch, but not master.

Reply:
ok, should I assume that we want the this PR based on development then?

Reply:
I'm not sure. @xsrust what do you think?

Reply:
@jollopre We had originally thought Master made more sense as it's more stable, but with Development diverging so far from it this sprint, it may make sense to rebase off development.

There's been work done to clean up the models so you'd be using the re-factored functions.
The downside is that there's still quite a few bugs on the dev branch at the moment, but we should be clearing those out soon, and as this feature is new, it shouldnt conflict with anything else.

Rebasing would also mean that Travis runs the up to date CI against the PRs.